Scripts/build.swift

```diff
@@ -131,10 +122,6 @@ guard let task = Task(rawValue: rawTask) else {
 	throw TaskError.code(1)
 }
 
-if task.shouldGenerateXcodeProject {
-	try execute(commandPath: "/usr/bin/swift", arguments: ["package", "generate-xcodeproj", "--output=generated/"])
```

Comment:
This is no longer a command in `swift package`, instead `xcodebuild` can be used without a project or workspace and Xcode will build the Swift package and link UIKit as needed